### PR TITLE
feat: Added FAQ List for an Event

### LINKED ISSUE #652: Add FAQ for Event
**Is it a ?**

**Feature Request**
    
**Description**

The API suports FAQ for event. This issue is aimed at implementing all the features of an FAQ listed [here](https://open-event-api.herokuapp.com/#faq)

- [x] FAQ List #667 
- [x] Create FAQ #677 
- [x] Delete FAQ #728 

**Would you like to work on the issue?**

Yes

### LINKED ISSUE #667: Show FAQ List
**Is it a ?**

- [ ] Bug Report

- [x] Feature Request
    
- [ ] Chore

**Parent Issue**
#652 
    
**Description**

There should be a layout to show list of all FAQ related to an event. API reference - [this](https://open-event-api.herokuapp.com/#faq-event-faq-collection-get) 

**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by Masquerade0097**
Fixes #667 

Changes: Loading FAQ list associated with an event through network, cache it for later use in the database, and load data selectively from network and database.

Screenshots for the change: 
fscr1

fscr2

### CODE REVIEW

**Comment by Masquerade0097** [comment]
@iamareebjamal I'm facing a problem here. When Faq object is stored in database the `event_id` field(which is the foreign key) in the `Faq` table referencing the `id` field in the `Event` table is storing null instead of the event id. I have specified the foreign key and relationship in the model. Please suggest changes.

File: app/src/main/java/org/fossasia/openevent/app/common/data/network/EventService.java [hunk 1]
```diff
@@ -82,4 +83,7 @@
 
     @GET("events/{id}/general-statistics")
     Observable<EventStatistics> getEventStatistics(@Path("id") long id);
+
+    @GET("/v1/events/{id}/faqs?include=event&fields[event]=id&page[size]=0")
```
Comment (iamareebjamal) [inline app/src/main/java/org/fossasia/openevent/app/common/data/network/EventService.java]: Should be `events/{id}/faqs?include=event&fields[event]=id&page[size]=0`

Comment (Masquerade0097) [reply app/src/main/java/org/fossasia/openevent/app/common/data/network/EventService.java]: Oops! just corrected it.

**Comment by iamareebjamal** [comment]
Please also add screenshots and create a follow up issue of creating a FAQ entry through the app

**Comment by Masquerade0097** [comment]
Yeah, sure.

**Comment by iamareebjamal** [comment]
Convert the UI to this form : 

Q: <question>
A: <Answer>

Make the question line bold. The indent of both items should be same

**Comment by Masquerade0097** [comment]
@iamareebjamal Please review.

**Pull request merged**